.circleci/config.yml

```diff
@@ -397,12 +400,12 @@ jobs:
       - <<: *couchbase
       - <<: *couchbase-setup
     <<: *shared
-    parallelism: 20
-    resource_class: large
+    parallelism: 15
```

Comment:
As discussed, we would like to run the main build a couple of times to check the time difference.
We either revert to 20 or keep 15 if the time difference is small.